internal/arenaskl/node.go

```diff
@@ -44,11 +44,12 @@ func (l *links) init(prevOffset, nextOffset uint32) {
 
 type node struct {
 	// Immutable fields, so no need to lock to access key.
-	keyOffset  uint32
-	keySize    uint32
-	keyTrailer uint64
-	valueSize  uint32
-	allocSize  uint32
+	keyOffset   uint32
+	keySize     uint32
+	keyTrailer  uint64
+	valueOffset uint32
```

Comment:
since now a node's memory may not be contiguous im not sure how to figure out where the start of a value will be (besides adding a `valueOffset` property, although does this slightly increase the size of a node?)